### ISSUE TITLE
raise exception in Util.py - 

### ISSUE THREAD

**Issue by bhargavasana** [MetropolitanTransportationCommission/fast-trips#11]
Was the standalone "raise" in the following piece of code intentional?
https://github.com/MetropolitanTransportationCommission/fast-trips/blob/develop/fasttrips/Util.py#L75

this results in an error
TypeError: exceptions must be old-style classes or derived from BaseException, not NoneType

Do we want to raise an exception?

**Comment by lmz** [comment]
I think it indicates there's a problem that we don't know how to handle, but it should throw a more useful exception than that.

**Comment by bhargavasana** [comment]
Right. I have asked Drew to take look at input which is creating this issue. The question is whether it should crash here or just write out a warning. Currently the issue is that it found a few stop ids that were in walk access file but not in the stops file.
